regression/CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ add_subdirectory(goto-cc-goto-analyzer)
 add_subdirectory(goto-diff)
 add_subdirectory(goto-instrument)
 add_subdirectory(goto-instrument-typedef)
+if(NOT WIN32)
+  add_subdirectory(goto-gcc)
```

Comment:
This will fail, as there is no CMakeLists.txt in `goto-gcc`. If you add a file with the following contents, it should work though:
```
add_test_pl_tests(
    "$<TARGET_FILE:goto-gcc>"
)
```

Reply:
Thank you for the guidance! I've added this in 4a5dff1 and will squash once CI succeeds.

Reply:
Think I got this wrong - if goto-gcc isn't a target, then we can't find its path in this way. As an alternative, I would hard-code the goto-gcc symlink location as `${CMAKE_BINARY_DIR}/goto-gcc`, and then use that path as the argument to `add_test_pl_tests`

Reply:
I've now gone for  cb9f586 - I feels like a horrible hack, but I haven't managed to figure out a way of marking a symlink as "executable". Maybe your hard-coded approach is the better option?